src/io.jl

```diff
@@ -293,14 +293,11 @@ function __prepare_buffer(x::AbstractArray{T,3}) where {T}
     if nchannels == 1
         ifelse(ndims(x) == 3, _prepare_buffer(dropdims(x, dims=3)), x)
     elseif nchannels == 2
-        convert(Array{GrayA}, colorview(GrayA, view(x, :, :, 1), view(x, :, :, 2)))
+        GrayA.(view(x, :, :, 1),view(x, :, :, 2))
     elseif nchannels == 3
-        convert(Array{RGB}, colorview(RGB, view(x, :, :, 1), view(x, :, :, 2), view(x, :, :, 3)))
+        RGB.(view(x, :, :, 1), view(x, :, :, 2), view(x, :, :, 3))
     elseif nchannels == 4
-        convert(
-            Array{RGBA},
-            colorview(RGBA, view(x, :, :, 1), view(x, :, :, 2), view(x, :, :, 3), view(x, :, :, 4)),
-        )
+        RGBA.(view(x, :, :, 1), view(x, :, :, 2), view(x, :, :, 3), view(x, :, :, 4))
```

Comment:
This is much nicer :+1:

Reply:
On the other hand, if we'll keep the ImageCore maybe the original version was a little bit faster:
```
julia> g(x) = convert(
                   Array{RGBA},
                   colorview(RGBA, view(x, :, :, 1), view(x, :, :, 2), view(x, :, :, 3), view(x, :, :, 4)),
              )
julia> f(x) = RGBA.(view(x, :, :, 1), view(x, :, :, 2), view(x, :, :, 3), view(x, :, :, 4))

julia> img = rand(N0f16, 2048, 2048, 4)

julia> @btime g(img);
  43.988 ms (8 allocations: 32.00 MiB)

julia> @btime f(img);
  52.528 ms (6 allocations: 32.00 MiB)

julia> img = rand(N0f16, 64, 64, 4)

julia> @btime g(img);
  39.093 μs (8 allocations: 32.41 KiB)

julia> @btime f(img);
  39.612 μs (6 allocations: 32.33 KiB)

```

Reply:
Thanks for testing that! Yeah, I'll revert given I've switched this PR to just removing Images.jl from the tests

Reply:
Although I have not benchmarked it, a simple `for` loop may be faster.